From e56269a62f8a272634042efbc8401e332d76b782 Mon Sep 17 00:00:00 2001
From: Jesper Juhl <jj@chaosbits.net>
Date: Fri, 24 Jun 2011 00:52:14 +0200
Patch-mainline: 3.1-4c4
Subject: [PATCH] Remove unneeded version.h include from drivers/staging/hv/hv_timesource.c

It was pointed out by 'make versioncheck' that include of
linux/version.h is not needed in drivers/staging/hv/hv_timesource.c .
This patch removes it.

Signed-off-by: Jesper Juhl <jj@chaosbits.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/hv_timesource.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/hv/hv_timesource.c b/drivers/staging/hv/hv_timesource.c
index 0efb049..2b0f9aa 100644
--- a/drivers/staging/hv/hv_timesource.c
+++ b/drivers/staging/hv/hv_timesource.c
@@ -22,7 +22,6 @@
  */
 #define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
 
-#include <linux/version.h>
 #include <linux/clocksource.h>
 #include <linux/init.h>
 #include <linux/module.h>
-- 
1.6.0.2

